projects/opentrain/gtfs_pipeline/heatmap/HeatMap.java

```diff
@@ -0,0 +1,500 @@
+          minData = heatMap[i][j];
+        }
+      }
+    }
+
+    // Load image to buffer
+    BufferedImage mapImage = null;
+    try {
+      mapImage = (ImageIO.read(new File(imagePath)));
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+
+    // Create image
+    final BufferedImage image =
+        new BufferedImage(mapImage.getWidth(), mapImage.getHeight(), BufferedImage.TYPE_INT_ARGB);
+    final Graphics2D graphics2D = image.createGraphics();
+
+    graphics2D.drawImage(mapImage, 0, 0, null);
+
+    // Draw the map
+    for (int i = 0; i < heatMap.length; i++) {
+      for (int j = 0; j < heatMap[0].length; j++) {
+        if (heatMap[i][j] == 0) {
+          System.out.println("ERROR:" + "\n" + "heatmap matrix value should not be 0");
+        }
+
+        float value = (float) (heatMap[i][j] - minData) / (float) (maxData - minData);
+
+        int r = (int) (((float) color2.getRed() - color1.getRed()) * value + color1.getRed());
+        int g = (int) (((float) color2.getGreen() - color1.getGreen()) * value + color1.getGreen());
+        int b = (int) (((float) color2.getBlue() - color1.getBlue()) * value + color1.getBlue());
+
+        Color color = new Color(r, g, b, 80);
+        graphics2D.setPaint(color);
+        if (inBorders(i, j)) {
+          graphics2D.fillRect(i, j, 2, 2);
+        }
+      }
+    }
+    for (StopsAround stopAround : stopsAroundList) {
+      graphics2D.setPaint(Color.black);
+      graphics2D.setFont(new Font("TimesRoman", Font.BOLD, 10));
+      int x = getXpixel(stopAround.stop.getStopLat(), stopAround.stop.getStopLon());
+      int y = getYpixel(stopAround.stop.getStopLat(), stopAround.stop.getStopLon());
+
+      graphics2D.drawString("X", x, y);
+    }
+
+    // TODO:Each stop to its place by adding/decreasing to x,y
+    graphics2D.drawString("Y", 330 + 80, 633 + 160);
+    graphics2D.dispose();
+    // Save file
+    boolean createImage = false;
+    try {
+      createImage = ImageIO.write(image, "png", new File(heatMapPath.get()));
+
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+    if (!createImage) {
+      System.out.println("can't create image");
+    }
+  }
+
+  public static void main(String[] args) throws IOException {
+    // TODO:
+    //      Boarders of israel,do not paint sea area
+    //      Improve x,y pixel method by using more data
+
+    Flags.parseCurrentPackage(args);
+    StopTime stopTime;
+    Trip trip;
+
+    try (FileInputStream tripInputStream =
+            new FileInputStream(new File("projects/opentrain/gtfs_pipeline/trips.protobin"));
+        FileInputStream stopTimeInputStream =
+            new FileInputStream(new File("projects/opentrain/gtfs_pipeline/stopTime.protobin"))) {
+
+      // Test some trip stops heatMap
+      trip = Trip.parseDelimitedFrom(tripInputStream);
+      stopTime = StopTime.parseDelimitedFrom(stopTimeInputStream);
+      if (trip == null) System.out.println("Error: trips file is empty");
```

Comment:
Please use `{}` for if statements.
Also, I think you should exit in this case, since this is a failure.